CODINGSTYLE.md

```diff
@@ -0,0 +1,56 @@
+# Coding Style
+
+## Naming
+
+1. Class names use CamelCase, for example: `class CallProfiler;`.
+   - Qt-related classes prepend an additional `R`, like `class RLine;`.
+     * A class is considered Qt-related if its parent class belongs to the Qt library.
+
+2. Variable names and using-declarations use snake_case:
+   - Example variable: `R3DWidget * viewer;`
+   - Example using-declaration: `using size_type = std::size_t;`
+
+3. Qt-related function names use CamelCase, while non-Qt-related function names use snake_case.
+   - Functions are Qt-related if they belong to a Qt-related class.
+   - Example Qt-related function: `QMdiSubWindow * RManager::addSubWindow(Args &&... args);`
+   - Example non-Qt-related function: `void initialize_buffer(pybind11::module & mod);`
```

Comment:
I distinquish and define the Qt-related classes and functions in the list.
Additionally, I reformat the file.